src/main/java/core/basesyntax/service/operation/impl/OperationReturnHandler.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.service.operation.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.operation.OperationHandler;
+
+public class OperationReturnHandler implements OperationHandler {
+
+    @Override
+    public void operationHandler(FruitTransaction fruitTransaction) {
+        int quantityFromDb = operationDao.get(fruitTransaction.getFruit());
```

Comment:
It seems that the `operationDao` is not defined in this class. You need to add a private field for the `operationDao` and initialize it via constructor.